src/Component/Menu/Navigation/Navigation.tsx

```diff
@@ -19,18 +19,19 @@ import config from 'shogunApplicationConfig';
 
 interface OwnProps {
   collapsed?: boolean;
+  entityConfigs?: any[];
```

Comment:
```suggestion
  entityConfigs?: GeneralEntityConfigType<BaseEntity>[];
```

(related imports should be added as well)